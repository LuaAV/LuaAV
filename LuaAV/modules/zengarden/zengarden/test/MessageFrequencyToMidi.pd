#N canvas 718 167 450 300 10;
#X obj 104 17 loadbang;
#X obj 104 237 print;
#X obj 173 17 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 139 99 -14.3;
#X msg 191 99 0;
#X obj 104 54 t b b b b b b;
#X obj 104 202 ftom;
#X msg 231 99 22 3 5;
#X msg 104 99 3.7;
#X connect 0 0 5 0;
#X connect 2 0 5 0;
#X connect 3 0 6 0;
#X connect 4 0 6 0;
#X connect 5 0 8 0;
#X connect 5 1 3 0;
#X connect 5 2 4 0;
#X connect 5 4 7 0;
#X connect 6 0 1 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
