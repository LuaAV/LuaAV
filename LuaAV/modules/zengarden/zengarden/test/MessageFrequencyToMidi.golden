[@ 0.000ms] print: 17.1369
[@ 0.000ms] print: -1500
[@ 0.000ms] print: -1500
[@ 0.000ms] print: -13.726
